packages/venia-ui/lib/components/CheckoutPage/ItemsReview/itemsReview.gql.js

```diff
@@ -4,7 +4,7 @@ import { ItemsReviewFragment } from './itemsReviewFragments.gql';
 
 const LIST_OF_PRODUCTS_IN_CART_QUERY = gql`
     query getItemsInCart($cartId: String!) {
-        cart(cart_id: $cartId) {
+        cart(cart_id: $cartId) @connection(key: "Cart") {
```

Comment:
Man, after seeing all these it would be great if we could enforce this somehow going forward. Guess we're just gonna have to keep a sharp eye during reviews.

Reply:
There may be some way to include this with the graphql schema check tool. I'll open an investigation ticket.

https://jira.corp.magento.com/browse/PWA-527